core/src/main/java/org/everit/json/schema/internal/DateTimeFormatValidator.java

```diff
@@ -27,23 +26,18 @@
         }
     }
 
-    private static final List<String> FORMATS_ACCEPTED = ImmutableList.of(
+    private static final List<String> FORMATS_ACCEPTED = Arrays.asList(
```

Comment:
`Arrays.asList()` returns a fixed-size but not immutable list. It would be better to wrap this into a `Collections.unmodifiableList()` too.